core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -396,8 +407,11 @@ public FileInfo setPath(String path) {
   }
 
   /**
-   * @param ufsPath the file UFS path to use
-   * @return the file information
+   * Sets the Under File Storage (UFS) path for the file this object holds information from
```

Comment:
- UFS means Under File System;
- Could simplify the phrase to be similar to `setPath`:
  - "path for this file information."